src/SSHDebugPS/AD7Port.cs

```diff
@@ -225,7 +225,14 @@ public bool IsLinux()
         public void Clean()
         {
             if (_connection != null)
-                _connection.Clean();
+            {
+                try
+                {
+                    _connection.Clean();
```

Comment:
can we make this the _connection?.Clean() notation?

Reply:
Ok